.github/workflows/zulip-ci.yml

```diff
@@ -4,7 +4,14 @@
 
 name: Zulip CI
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
```

Comment:
Should we add tags to this as well? I think we've sometimes pushed a tag for a release candidate or something that isn't on the `main` branch, and while we'll push a branch for it, I think those branches likely won't have any of these naming conventions.

(I don't remember how we name those).

Reply:
Sure, added all tags.